taker-frontend/src/components/Trade.tsx

```diff
@@ -289,6 +297,7 @@ function Quantity({ min, max, onChange, quantity }: QuantityProps) {
                     min={min}
                     max={max}
                     default={min}
+                    step={100}
```

Comment:
You could use the same variable here as well: 
```suggestion
                    step={quantityIncrement}
```